st2common/bin/paramiko_ssh_evenlets_tester.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python2.7
```

Comment:
I could be convinced not to check in this file but this helps stand alone test the SSH part alone without spinning up st2. Turned out pretty useful when I was developing.